index.html

```diff
@@ -147,40 +169,57 @@ <h2>Uploading New Materials</h2>
+				<div class="picture">
+					<img src="./Resources/CLARK-Children-Objects.png" width="1000px">
+					<p style="padding-top: 20px;">Keep in mind only learning objects you have created can be a member of a hierarchy.  You can edit an object&#39;s children at anytime by simply navigating to your dashboard and selecting the learning object you would like to make a revision on from the <i>Released</i> tab.</p>
+				</div>
+			</div>
+			<br><br>
+
 			<a name="Editing"><div class="section">
 				<h2>Editing Learning Objects</h2>
 				<hr>
-				<p>Go back to your learning objects dashboard. You will notice that the learning object you have just created has appeared in this list.</p>
-				<p style="padding-bottom: 20px;">To edit a learning object, click the 'meatball menu' (...) of the object you want to edit. There will be a list of different options, including an option to edit the learning object itself and manage the materials in it. Note: if your object is currently waiting or under review, you cannot access the meatball menu and, therefore, cannot edit your object.</p>
+				<p>Navigate to your dashboard. By default, you should see your draft learning objects under the <i>Draft</i> tab. You will find all learning objects that you have created that have not yet been released in this list.</p>
+				<p style="padding-bottom: 20px;">To edit a learning object, click the 'meatball menu' (...) of the object you want to edit. There will be a list of different options, including an option to edit the learning object itself and manage the materials in it. From here, click the <i>Edit</i> option and you will be redirected to the learning object creation page. While making changes in the learning object builder, your work will save automatically, similarly to Google Documents. This means you may also leave and return to work on edits at a later time.</p>
 				<div class="picture">
-					<img src="./Resources/CLARK-Editing-Options.png" width="1000px">
-					<p style="padding-top: 20px;">From here, click the <i>Edit</i> option and you will be redirected to the learning object creation page. You can edit or change as much as you desire. The learning object builder automatically saves your work, similar to a Google Document.</p>
+					<img src="./Resources/CLARK-Draft-Edit.png" width="1000px">
+					<p style="padding-top: 20px;">Note: you can only edit a learning object that is in an unreleased state. You will not have the option to edit learning objects that are waiting for review, under review, in proofing, or denied.</p>
```

Comment:
In addition to this: "If you want to edit an in-review object, you must unsubmit the object first.  This can be done in the meatball menu options."